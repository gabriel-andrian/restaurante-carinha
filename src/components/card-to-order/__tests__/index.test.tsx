import React from 'react';
import CardToOrder from '../index';
import renderer from 'react-test-renderer';

jest.mock('react-router-dom', () => ({
  useRouteMatch: jest.fn(() => ({ url: '/url' })),
  Link: jest.fn(({ to, children }) => <a href={to}>{children}</a>),
}));

describe('card-to-order', () => {
  it('renders', () => {
    const tree = renderer
      .create(
        <CardToOrder
          title={'Título de Teste'}
          description={'Descrição de teste'}
          img={'#'}
          price={0}
          id={1}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
