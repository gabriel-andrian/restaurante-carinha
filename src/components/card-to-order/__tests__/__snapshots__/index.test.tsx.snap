// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card-to-order renders 1`] = `
<div
  className="sc-bdnylx fUrupC"
>
  <a
    href="/url/1"
  >
    <div
      className="sc-dlnjPT egUXgI"
    >
      <div
        className="sc-hKFyIo cvSBHa"
      />
      <div
        className="sc-eCApGN fMQorY"
      >
        <div
          className="sc-jSFkmK gpilit"
        >
          Título de Teste
        </div>
        <div
          className="sc-gKAblj itjylz"
        >
          Descrição de teste
        </div>
        <div
          className="sc-iCoHVE izPhX"
        >
          R$ 0.00
        </div>
      </div>
    </div>
  </a>
</div>
`;
